modules/tinymce/src/core/main/ts/newline/InsertBlock.ts

```diff
@@ -235,6 +242,15 @@ const addBrToBlockIfNeeded = (dom, block) => {
   }
 };
 
+const shouldEndContainer = (editor: Editor, container: Node) => {
```

Comment:
```suggestion
const shouldEndContainer = (editor: Editor, container: Node | undefined) => {
```
This should be this if we have the nullable check and looking at how it's defined, it looks like the case we need to protect against is `undefined`.